app/src/main/java/org/thoughtcrime/securesms/conversation/v2/ConversationActivityV2.kt

```diff
@@ -1528,14 +1537,11 @@ class ConversationActivityV2 : PassphraseRequiredActionBarActivity(), InputBarDe
         sendEmojiRemoval(emoji, message)
     }
 
+    /**
```

Comment:
Could just be a single line comment via `//`

Reply:
This is the format to declare the description of the method. This way it shows up on docs or when looking up methods

Reply:
Okey-doke. Maybe we should be using that throughout then. Happy to leave it for now.